src/themes/creator.js

```diff
@@ -497,14 +545,16 @@ export default function createTheme(
       scale300: '8px',
       scale400: '10px',
       scale500: '12px',
-      scale550: '14px',
-      scale600: '16px',
-      scale700: '20px',
-      scale800: '24px',
-      scale900: '32px',
-      scale1000: '40px',
-      scale1200: '48px',
-      scale1400: '56px',
+      scale600: '14px',
```

Comment:
if we go down on this path, and change values for these props, we should probably write a migration guide, as folks depend on these values a lot for overrides / Block props

Reply:
it may be straightforward to write a codemod for this.

- look for strings in files that import baseui block. 
- if strings match, replace with new scale title. (may have false-positives)
- search for baseui components with overrides prop. replace scales from the props function (this is likely tricky, but we could have high confidence with unit tests)